kuma/static/styles/components/content.styl

```diff
@@ -7,6 +7,13 @@
 
 $table-blue = #d4dde4;
 
+/*
+    Used to draw borders around objects which should only be bordered in
+    the editor.
+*/
+
+$editor-box-border-style = solid 1px #6699FF40;
```

Comment:
Let's move this variable into kuma/styles/includes/vars.styl so we can use it everywhere!

Remove 'style' from the variable name (I don't want to imply the value of the variable is just the style).

Border short hand should be in the order width, style, colour.

Please use a hex or rgba colour format.